webui/assets/unsuspend.html

```diff
@@ -26,4 +26,10 @@
   </form>
 </body>
 
-{{ template "footer.html" }}
```

Comment:
❓ Are you sure we can not pass variables to the `template` function?
It seems like defeating the purpose.